packages/node_modules/@ciscospark/internal-plugin-conversation/src/conversation.js

```diff
@@ -873,8 +874,8 @@ const Conversation = SparkPlugin.extend({
       user: params.participants[1]
     }), Object.assign(options, {includeConvWithDeletedUserUUID: true, includeParticipants: true}))
       .then((conversation) => {
-        if (params.comment) {
-          return this.post(conversation, {displayName: params.comment})
+        if (params.comment || params.markdown) {
+          return this.post(conversation, {displayName: params.comment, content: params.markdown})
```

Comment:
I think this'll fail eslint.

Reply:
hmm i'm not seeing any errors for this line in eslint. what should it be?